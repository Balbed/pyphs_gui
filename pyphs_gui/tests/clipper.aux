\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}System netlist}{1}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Core dimensions}{1}{section.2}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces  Graph of system \texttt  {clipper}. }}{2}{figure.1}}
\newlabel{fig:graphclipper}{{1}{2}{Graph of system \texttt {clipper}}{figure.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Core quantities}{3}{section.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}Core constants}{3}{subsection.3.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}Core variables}{3}{subsection.3.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.3}Core inputs}{4}{subsection.3.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.4}Core outputs}{4}{subsection.3.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.5}Core connectors}{4}{subsection.3.5}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Core constitutive relations}{5}{section.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}Core storage function}{5}{subsection.4.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.2}Core dissipation function}{5}{subsection.4.2}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Core structure}{6}{section.5}}
\@writefile{toc}{\contentsline {subsection}{\numberline {5.1}Core M-structure}{6}{subsection.5.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {5.2}Core J-structure}{7}{subsection.5.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {5.3}Core R-structure}{8}{subsection.5.3}}
