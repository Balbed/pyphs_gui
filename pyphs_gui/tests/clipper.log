This is pdfTeX, Version 3.14159265-2.6-1.40.18 (TeX Live 2017) (preloaded format=pdflatex 2017.8.10)  14 NOV 2017 20:15
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**clipper.tex
(./clipper.tex
LaTeX2e <2017-04-15>
Babel <3.10> and hyphenation patterns for 84 language(s) loaded.
(/usr/local/texlive/2017/texmf-dist/tex/latex/base/article.cls
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
(/usr/local/texlive/2017/texmf-dist/tex/latex/base/size11.clo
File: size11.clo 2014/09/29 v1.4h Standard LaTeX file (size option)
)
\c@part=\count79
\c@section=\count80
\c@subsection=\count81
\c@subsubsection=\count82
\c@paragraph=\count83
\c@subparagraph=\count84
\c@figure=\count85
\c@table=\count86
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
)
(/usr/local/texlive/2017/texmf-dist/tex/latex/geometry/geometry.sty
Package: geometry 2010/09/12 v5.6 Page Geometry

(/usr/local/texlive/2017/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2014/10/28 v1.15 key=value parser (DPC)
\KV@toks@=\toks14
)
(/usr/local/texlive/2017/texmf-dist/tex/generic/oberdiek/ifpdf.sty
Package: ifpdf 2017/03/15 v3.2 Provides the ifpdf switch
)
(/usr/local/texlive/2017/texmf-dist/tex/generic/oberdiek/ifvtex.sty
Package: ifvtex 2016/05/16 v1.6 Detect VTeX and its facilities (HO)
Package ifvtex Info: VTeX not detected.
)
(/usr/local/texlive/2017/texmf-dist/tex/generic/ifxetex/ifxetex.sty
Package: ifxetex 2010/09/12 v0.6 Provides ifxetex conditional
)
\Gm@cnth=\count87
\Gm@cntv=\count88
\c@Gm@tempcnt=\count89
\Gm@bindingoffset=\dimen103
\Gm@wd@mp=\dimen104
\Gm@odd@mp=\dimen105
\Gm@even@mp=\dimen106
\Gm@layoutwidth=\dimen107
\Gm@layoutheight=\dimen108
\Gm@layouthoffset=\dimen109
\Gm@layoutvoffset=\dimen110
\Gm@dimlist=\toks15
)
(/usr/local/texlive/2017/texmf-dist/tex/latex/parskip/parskip.sty
Package: parskip 2001/04/09 non-zero parskip adjustments
)
(/usr/local/texlive/2017/texmf-dist/tex/latex/graphics/graphicx.sty
Package: graphicx 2014/10/28 v1.0g Enhanced LaTeX Graphics (DPC,SPQR)

(/usr/local/texlive/2017/texmf-dist/tex/latex/graphics/graphics.sty
Package: graphics 2017/04/14 v1.1b Standard LaTeX Graphics (DPC,SPQR)

(/usr/local/texlive/2017/texmf-dist/tex/latex/graphics/trig.sty
Package: trig 2016/01/03 v1.10 sin cos tan (DPC)
)
(/usr/local/texlive/2017/texmf-dist/tex/latex/graphics-cfg/graphics.cfg
File: graphics.cfg 2016/06/04 v1.11 sample graphics configuration
)
Package graphics Info: Driver file: pdftex.def on input line 99.

(/usr/local/texlive/2017/texmf-dist/tex/latex/graphics-def/pdftex.def
File: pdftex.def 2017/01/12 v0.06k Graphics/color for pdfTeX

(/usr/local/texlive/2017/texmf-dist/tex/generic/oberdiek/infwarerr.sty
Package: infwarerr 2016/05/16 v1.4 Providing info/warning/error messages (HO)
)
(/usr/local/texlive/2017/texmf-dist/tex/generic/oberdiek/ltxcmds.sty
Package: ltxcmds 2016/05/16 v1.23 LaTeX kernel commands for general use (HO)
)
\Gread@gobject=\count90
))
\Gin@req@height=\dimen111
\Gin@req@width=\dimen112
)
(/usr/local/texlive/2017/texmf-dist/tex/latex/amsfonts/amssymb.sty
Package: amssymb 2013/01/14 v3.01 AMS font symbols

(/usr/local/texlive/2017/texmf-dist/tex/latex/amsfonts/amsfonts.sty
Package: amsfonts 2013/01/14 v3.01 Basic AMSFonts support
\@emptytoks=\toks16
\symAMSa=\mathgroup4
\symAMSb=\mathgroup5
LaTeX Font Info:    Overwriting math alphabet `\mathfrak' in version `bold'
(Font)                  U/euf/m/n --> U/euf/b/n on input line 106.
))
(/usr/local/texlive/2017/texmf-dist/tex/latex/preprint/authblk.sty
Package: authblk 2001/02/27 1.3 (PWD)
\affilsep=\skip43
\@affilsep=\skip44
\c@Maxaffil=\count91
\c@authors=\count92
\c@affil=\count93
)
(/usr/local/texlive/2017/texmf-dist/tex/latex/hyperref/hyperref.sty
Package: hyperref 2017/03/14 v6.85a Hypertext links for LaTeX

(/usr/local/texlive/2017/texmf-dist/tex/generic/oberdiek/hobsub-hyperref.sty
Package: hobsub-hyperref 2016/05/16 v1.14 Bundle oberdiek, subset hyperref (HO)


(/usr/local/texlive/2017/texmf-dist/tex/generic/oberdiek/hobsub-generic.sty
Package: hobsub-generic 2016/05/16 v1.14 Bundle oberdiek, subset generic (HO)
Package: hobsub 2016/05/16 v1.14 Construct package bundles (HO)
Package hobsub Info: Skipping package `infwarerr' (already loaded).
Package hobsub Info: Skipping package `ltxcmds' (already loaded).
Package: ifluatex 2016/05/16 v1.4 Provides the ifluatex switch (HO)
Package ifluatex Info: LuaTeX not detected.
Package hobsub Info: Skipping package `ifvtex' (already loaded).
Package: intcalc 2016/05/16 v1.2 Expandable calculations with integers (HO)
Package hobsub Info: Skipping package `ifpdf' (already loaded).
Package: etexcmds 2016/05/16 v1.6 Avoid name clashes with e-TeX commands (HO)
Package etexcmds Info: Could not find \expanded.
(etexcmds)             That can mean that you are not using pdfTeX 1.50 or
(etexcmds)             that some package has redefined \expanded.
(etexcmds)             In the latter case, load this package earlier.
Package: kvsetkeys 2016/05/16 v1.17 Key value parser (HO)
Package: kvdefinekeys 2016/05/16 v1.4 Define keys (HO)
Package: pdftexcmds 2017/03/19 v0.25 Utility functions of pdfTeX for LuaTeX (HO
)
Package pdftexcmds Info: LuaTeX not detected.
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
Package: pdfescape 2016/05/16 v1.14 Implements pdfTeX's escape features (HO)
Package: bigintcalc 2016/05/16 v1.4 Expandable calculations on big integers (HO
)
Package: bitset 2016/05/16 v1.2 Handle bit-vector datatype (HO)
Package: uniquecounter 2016/05/16 v1.3 Provide unlimited unique counter (HO)
)
Package hobsub Info: Skipping package `hobsub' (already loaded).
Package: letltxmacro 2016/05/16 v1.5 Let assignment for LaTeX macros (HO)
Package: hopatch 2016/05/16 v1.3 Wrapper for package hooks (HO)
Package: xcolor-patch 2016/05/16 xcolor patch
Package: atveryend 2016/05/16 v1.9 Hooks at the very end of document (HO)
Package atveryend Info: \enddocument detected (standard20110627).
Package: atbegshi 2016/06/09 v1.18 At begin shipout hook (HO)
Package: refcount 2016/05/16 v3.5 Data extraction from label references (HO)
Package: hycolor 2016/05/16 v1.8 Color options for hyperref/bookmark (HO)
)
(/usr/local/texlive/2017/texmf-dist/tex/latex/oberdiek/auxhook.sty
Package: auxhook 2016/05/16 v1.4 Hooks for auxiliary files (HO)
)
(/usr/local/texlive/2017/texmf-dist/tex/latex/oberdiek/kvoptions.sty
Package: kvoptions 2016/05/16 v3.12 Key value format for package options (HO)
)
\@linkdim=\dimen113
\Hy@linkcounter=\count94
\Hy@pagecounter=\count95

(/usr/local/texlive/2017/texmf-dist/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2017/03/14 v6.85a Hyperref: PDFDocEncoding definition (HO)
)
\Hy@SavedSpaceFactor=\count96

(/usr/local/texlive/2017/texmf-dist/tex/latex/latexconfig/hyperref.cfg
File: hyperref.cfg 2002/06/06 v1.2 hyperref configuration of TeXLive
)
Package hyperref Info: Hyper figures OFF on input line 4498.
Package hyperref Info: Link nesting OFF on input line 4503.
Package hyperref Info: Hyper index ON on input line 4506.
Package hyperref Info: Plain pages OFF on input line 4513.
Package hyperref Info: Backreferencing OFF on input line 4518.
Package hyperref Info: Implicit mode ON; LaTeX internals redefined.
Package hyperref Info: Bookmarks ON on input line 4751.
\c@Hy@tempcnt=\count97

(/usr/local/texlive/2017/texmf-dist/tex/latex/url/url.sty
\Urlmuskip=\muskip10
Package: url 2013/09/16  ver 3.4  Verb mode for urls, etc.
)
LaTeX Info: Redefining \url on input line 5104.
\XeTeXLinkMargin=\dimen114
\Fld@menulength=\count98
\Field@Width=\dimen115
\Fld@charsize=\dimen116
Package hyperref Info: Hyper figures OFF on input line 6358.
Package hyperref Info: Link nesting OFF on input line 6363.
Package hyperref Info: Hyper index ON on input line 6366.
Package hyperref Info: backreferencing OFF on input line 6373.
Package hyperref Info: Link coloring OFF on input line 6378.
Package hyperref Info: Link coloring with OCG OFF on input line 6383.
Package hyperref Info: PDF/A mode OFF on input line 6388.
LaTeX Info: Redefining \ref on input line 6428.
LaTeX Info: Redefining \pageref on input line 6432.
\Hy@abspage=\count99
\c@Item=\count100
\c@Hfootnote=\count101
)

Package hyperref Message: Driver (autodetected): hpdftex.

(/usr/local/texlive/2017/texmf-dist/tex/latex/hyperref/hpdftex.def
File: hpdftex.def 2017/03/14 v6.85a Hyperref driver for pdfTeX
\Fld@listcount=\count102
\c@bookmark@seq@number=\count103

(/usr/local/texlive/2017/texmf-dist/tex/latex/oberdiek/rerunfilecheck.sty
Package: rerunfilecheck 2016/05/16 v1.8 Rerun checks for auxiliary files (HO)
Package uniquecounter Info: New unique counter `rerunfilecheck' on input line 2
82.
)
\Hy@SectionHShift=\skip45
)
(./clipper.aux)
\openout1 = `clipper.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 22.
LaTeX Font Info:    ... okay on input line 22.

*geometry* driver: auto-detecting
*geometry* detected driver: pdftex
*geometry* verbose mode - [ preamble ] result:
* driver: pdftex
* paper: letterpaper
* layout: <same size as paper>
* layoutoffset:(h,v)=(0.0pt,0.0pt)
* modes: 
* h-part:(L,W,R)=(92.14519pt, 430.00462pt, 92.14519pt)
* v-part:(T,H,B)=(95.39737pt, 556.47656pt, 143.09605pt)
* \paperwidth=614.295pt
* \paperheight=794.96999pt
* \textwidth=430.00462pt
* \textheight=556.47656pt
* \oddsidemargin=19.8752pt
* \evensidemargin=19.8752pt
* \topmargin=-13.87262pt
* \headheight=12.0pt
* \headsep=25.0pt
* \topskip=11.0pt
* \footskip=30.0pt
* \marginparwidth=59.0pt
* \marginparsep=10.0pt
* \columnsep=10.0pt
* \skip\footins=10.0pt plus 4.0pt minus 2.0pt
* \hoffset=0.0pt
* \voffset=0.0pt
* \mag=1000
* \@twocolumnfalse
* \@twosidefalse
* \@mparswitchfalse
* \@reversemarginfalse
* (1in=72.27pt=25.4mm, 1cm=28.453pt)

(/usr/local/texlive/2017/texmf-dist/tex/context/base/mkii/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
\scratchcounter=\count104
\scratchdimen=\dimen117
\scratchbox=\box26
\nofMPsegments=\count105
\nofMParguments=\count106
\everyMPshowfont=\toks17
\MPscratchCnt=\count107
\MPscratchDim=\dimen118
\MPnumerator=\count108
\makeMPintoPDFobject=\count109
\everyMPtoPDFconversion=\toks18
) (/usr/local/texlive/2017/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty
Package: epstopdf-base 2016/05/15 v2.6 Base part for package epstopdf

(/usr/local/texlive/2017/texmf-dist/tex/latex/oberdiek/grfext.sty
Package: grfext 2016/05/16 v1.2 Manage graphics extensions (HO)
)
Package epstopdf-base Info: Redefining graphics rule for `.eps' on input line 4
38.
Package grfext Info: Graphics extension search list:
(grfext)             [.png,.pdf,.jpg,.mps,.jpeg,.jbig2,.jb2,.PNG,.PDF,.JPG,.JPE
G,.JBIG2,.JB2,.eps]
(grfext)             \AppendGraphicsExtensions on input line 456.

(/usr/local/texlive/2017/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg
File: epstopdf-sys.cfg 2010/07/13 v1.3 Configuration of (r)epstopdf for TeX Liv
e
))
\AtBeginShipoutBox=\box27
Package hyperref Info: Link coloring OFF on input line 22.

(/usr/local/texlive/2017/texmf-dist/tex/latex/hyperref/nameref.sty
Package: nameref 2016/05/21 v2.44 Cross-referencing by name of section

(/usr/local/texlive/2017/texmf-dist/tex/generic/oberdiek/gettitlestring.sty
Package: gettitlestring 2016/05/16 v1.5 Cleanup title references (HO)
)
\c@section@level=\count110
)
LaTeX Info: Redefining \ref on input line 22.
LaTeX Info: Redefining \pageref on input line 22.
LaTeX Info: Redefining \nameref on input line 22.

(./clipper.out) (./clipper.out)
\@outlinefile=\write3
\openout3 = `clipper.out'.

LaTeX Font Info:    Try loading font information for U+msa on input line 27.

(/usr/local/texlive/2017/texmf-dist/tex/latex/amsfonts/umsa.fd
File: umsa.fd 2013/01/14 v3.01 AMS symbols A
)
LaTeX Font Info:    Try loading font information for U+msb on input line 27.

(/usr/local/texlive/2017/texmf-dist/tex/latex/amsfonts/umsb.fd
File: umsb.fd 2013/01/14 v3.01 AMS symbols B
)
LaTeX Font Info:    Try loading font information for OMS+cmr on input line 27.

(/usr/local/texlive/2017/texmf-dist/tex/latex/base/omscmr.fd
File: omscmr.fd 2014/09/29 v2.5h Standard LaTeX font definitions
)
LaTeX Font Info:    Font shape `OMS/cmr/m/n' in size <8> not available
(Font)              Font shape `OMS/cmsy/m/n' tried instead on input line 27.
LaTeX Font Info:    Font shape `OMS/cmr/m/n' in size <6> not available
(Font)              Font shape `OMS/cmsy/m/n' tried instead on input line 27.

Overfull \hbox (109.88925pt too wide) in paragraph at lines 27--27
[][][]$\OT1/cmtt/m/n/9 https : / / www . ircam . fr / recherche / equipes-[]rec
herche / systemes-[]et-[]signaux-[]sonores-[]audioacoustique-[]instruments-[]s3
am/$[]| 
 []


Overfull \hbox (6.50426pt too wide) in paragraph at lines 92--94
 [] 
 []


</Users/afalaize/Developement/repos/pyphs_gui/pyphs_gui/clipper.pdf, id=66, 462
.52798pt x 346.89601pt>
File: /Users/afalaize/Developement/repos/pyphs_gui/pyphs_gui/clipper.pdf Graphi
c file (type pdf)

<use /Users/afalaize/Developement/repos/pyphs_gui/pyphs_gui/clipper.pdf>
Package pdftex.def Info: /Users/afalaize/Developement/repos/pyphs_gui/pyphs_gui
/clipper.pdf used on input line 101.
(pdftex.def)             Requested size: 430.00462pt x 322.51996pt.


LaTeX Warning: `!h' float specifier changed to `!ht'.

[1

{/usr/local/texlive/2017/texmf-var/fonts/map/pdftex/updmap/pdftex.map}]
[2 </Users/afalaize/Developement/repos/pyphs_gui/pyphs_gui/clipper.pdf>]
LaTeX Font Info:    Font shape `OMS/cmr/m/n' in size <10.95> not available
(Font)              Font shape `OMS/cmsy/m/n' tried instead on input line 155.

[3] [4] [5] [6] [7] [8]
Package atveryend Info: Empty hook `BeforeClearDocument' on input line 487.
 [9]
Package atveryend Info: Empty hook `AfterLastShipout' on input line 487.
 (./clipper.aux)
Package atveryend Info: Executing hook `AtVeryEndDocument' on input line 487.
Package atveryend Info: Executing hook `AtEndAfterFileList' on input line 487.
Package rerunfilecheck Info: File `clipper.out' has not changed.
(rerunfilecheck)             Checksum: 23BBC6E316B8D474EBBEA113184410FB;916.
Package atveryend Info: Empty hook `AtVeryVeryEnd' on input line 487.
 ) 
Here is how much of TeX's memory you used:
 5985 strings out of 492995
 88315 string characters out of 6132704
 185519 words of memory out of 5000000
 9447 multiletter control sequences out of 15000+600000
 15145 words of font info for 58 fonts, out of 8000000 for 9000
 1141 hyphenation exceptions out of 8191
 32i,15n,28p,1070b,373s stack positions out of 5000i,500n,10000p,200000b,80000s
</usr/local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx10.pfb
></usr/local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx12.pfb>
</usr/local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx6.pfb></
usr/local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx8.pfb></us
r/local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmcsc10.pfb></us
r/local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmex10.pfb></usr
/local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi10.pfb></usr/
local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi6.pfb></usr/lo
cal/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi8.pfb></usr/loca
l/texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb></usr/local/
texlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmr12.pfb></usr/local/te
xlive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmr17.pfb></usr/local/texl
ive/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmr6.pfb></usr/local/texlive
/2017/texmf-dist/fonts/type1/public/amsfonts/cm/cmr8.pfb></usr/local/texlive/20
17/texmf-dist/fonts/type1/public/amsfonts/cm/cmsy10.pfb></usr/local/texlive/201
7/texmf-dist/fonts/type1/public/amsfonts/cm/cmsy6.pfb></usr/local/texlive/2017/
texmf-dist/fonts/type1/public/amsfonts/cm/cmsy8.pfb></usr/local/texlive/2017/te
xmf-dist/fonts/type1/public/amsfonts/cm/cmti10.pfb></usr/local/texlive/2017/tex
mf-dist/fonts/type1/public/amsfonts/cm/cmtt10.pfb></usr/local/texlive/2017/texm
f-dist/fonts/type1/public/amsfonts/cm/cmtt9.pfb></usr/local/texlive/2017/texmf-
dist/fonts/type1/public/amsfonts/symbols/msam10.pfb></usr/local/texlive/2017/te
xmf-dist/fonts/type1/public/amsfonts/symbols/msbm10.pfb>
Output written on clipper.pdf (9 pages, 246153 bytes).
PDF statistics:
 233 PDF objects out of 1000 (max. 8388607)
 176 compressed objects within 2 object streams
 26 named destinations out of 1000 (max. 500000)
 126 words of extra memory for PDF output out of 10000 (max. 10000000)

