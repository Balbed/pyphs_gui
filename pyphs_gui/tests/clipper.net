electronics.source input ('#', 'A'): type=voltage;
electronics.capacitor C ('A', 'C'): C=('Csymbol', 1e-09);
electronics.source output ('B', '#'): type=current;
electronics.diode d2 ('#', 'B'): Is=('Is', 2e-09); R=('Rd', 0.5); v0=('v0', 0.026); mu=('mu', 1.7);