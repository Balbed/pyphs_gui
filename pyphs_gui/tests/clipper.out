\BOOKMARK [1][-]{section.1}{System netlist}{}% 1
\BOOKMARK [1][-]{section.2}{Core dimensions}{}% 2
\BOOKMARK [1][-]{section.3}{Core quantities}{}% 3
\BOOKMARK [2][-]{subsection.3.1}{Core constants}{section.3}% 4
\BOOKMARK [2][-]{subsection.3.2}{Core variables}{section.3}% 5
\BOOKMARK [2][-]{subsection.3.3}{Core inputs}{section.3}% 6
\BOOKMARK [2][-]{subsection.3.4}{Core outputs}{section.3}% 7
\BOOKMARK [2][-]{subsection.3.5}{Core connectors}{section.3}% 8
\BOOKMARK [1][-]{section.4}{Core constitutive relations}{}% 9
\BOOKMARK [2][-]{subsection.4.1}{Core storage function}{section.4}% 10
\BOOKMARK [2][-]{subsection.4.2}{Core dissipation function}{section.4}% 11
\BOOKMARK [1][-]{section.5}{Core structure}{}% 12
\BOOKMARK [2][-]{subsection.5.1}{Core M-structure}{section.5}% 13
\BOOKMARK [2][-]{subsection.5.2}{Core J-structure}{section.5}% 14
\BOOKMARK [2][-]{subsection.5.3}{Core R-structure}{section.5}% 15
